### ISSUE TITLE
Tests have shared state

### ISSUE THREAD

**Issue by OrdonTeam** [elpassion/MiniSpek#1]
See: https://github.com/elpassion/MiniSpek/commit/308a8720c690dfd2fde7b6bd6ae3448967cca6e4

**Comment by ghostbuster91** [comment]
Yes, it is easy to declare variable that will be shared between testes, on
the other hand it is also easy to declare variable that won't be shared.
Just put them inside mspek lambda.

**Comment by OrdonTeam** [comment]
It cannot be used in methods then
